src/logs.mli

```diff
@@ -0,0 +1,19 @@
+(******************************************************************************)
+(*                              ocplib-simplex                                *)
+(*                                                                            *)
+(* Copyright (C) --- OCamlPro --- See README.md for information and licensing *)
+(******************************************************************************)
+
+(** The module charged of the library feedbacks.
+    Logs can be activated by changing the environment variable
+    `OCPLIB_SIMPLEX_DEBUG` *)
+
+(** The value of OCPLIB_SIMPLEX_DEBUG as an integer; by default to `0`
+    if not set or set to something else than an integer. *)
+val debug_level : int
```

Comment:
Maybe it'd be time to convert this to use `logs` instead of using a custom implementation ?

Reply:
For our needs I thought adding a dependency would be over-engineering things a little bit. `logs` is quite powerful and here we do not really need all the features it provides.

Reply:
Well, we need logging as a lib: i.e. logging that can be turned on/off depending on the situation, and if every library has its own mechanism for that, it's hell for users, so since there is now a quite popular framework for doing that (and which has been designed to be as efficient as possible), why not use it ?

Reply:
I guess this is just personal taste, but I will not defend my opinion on the subject ^^ I'll switch to `logs`